### DISCUSSION TITLE
UVtools forLinux downloaded and unzipped, now what?

### DISCUSSION THREAD

**Discussion by BootZ123** [sn4k3/UVtools#284] [Q&A] [answered]
Linux file Downloaded and unzipped, now what?
I looked through the folder to see if there was a file to start up the program.
I see a file called "UVTools" but only asked for a program to open it in.
What now?

**Comment by sn4k3** [comment, marked as answer]
To run UVtools open it folder on a terminal and call one of:

- ./UVtools
- sh UVtools.sh

As a pratical alternative you can create a shortcut on Desktop

**Reply by BootZ123** [reply]
I opened UVTools directory in a Terminal Window.
Attempted both commands in Terminal without any luck.
Am I missing any packages or dependencies for UVTools?

**Reply by sn4k3** [reply]
What is your OS and version?

**Reply by BootZ123** [reply]
Linux Mint.

**Reply by sn4k3** [reply]
You need these dependencies: https://github.com/sn4k3/UVtools#ubuntumintdebiansimilars
